api/genomics/wdl/IngestSyntheticVcfWf.wdl

```diff
@@ -2,40 +2,92 @@ version 1.0
 
 workflow IngestSyntheticVcfWf {
   input {
-    File base_vcf
+    File base_vcf # Sample VCF file that the randomization will be based on
     File base_vcf_index
-    File file_of_sample_names
-    String output_file
+    File sample_names # Newline separated list of names 
+    Int num_partitions # sample_names will be evenly split across this #
+    String output_bucket # Bucket that will contain the generated TSV data
+    File probe_info_file
+
+    Int batch_size # Should be set to 10 for actual runs
```

Comment:
If there's an actual limit of 4,000 on this please include that in the comment even if it should be set to 10 for actual runs